src/main/java/org/jabref/model/entry/field/FieldProperty.java

```diff
@@ -18,7 +18,7 @@ public enum FieldProperty {
     IDENTIFIER,
 
     LANGUAGE,
-
+    LANGUAGEID,
     // Field content is text, but should be interpreted as markdown
```

Comment:
Please keep empty lines as separator

Reply:
Maybe, this and `LANGUAGE` can just be removed. These properties are not used.

Alternataively, use it. And then use the langId field editor as you introduced for all fields having this property.

Reply:
I have removed `LANGUAGEID` and instead using `LANGUAGE` for langid field editor/view model. I am now assuming that the language IDs given in the biblatex doc are also all the supported languages.